src/pages/ServiceList/__tests__/ServiceListComponent.test.ts

```diff
@@ -78,20 +78,18 @@ describe('ServiceListContainer#sortServices', () => {
   it('should sort ascending', done => {
     const sortField = ServiceListFilters.sortFields.find(s => s.title === 'Service Name')!;
     const services = [makeService('A', {}, {}), makeService('B', {}, {})];
-    ServiceListFilters.sortServices(services, sortField, true).then(sorted => {
-      expect(sorted[0].name).toBe('A');
-      expect(sorted[1].name).toBe('B');
-      done();
-    });
+    const sorted = ServiceListFilters.sortServices(services, sortField, true);
+    expect(sorted[0].name).toBe('A');
+    expect(sorted[1].name).toBe('B');
+    done();
```

Comment:
Since these tests no longer deal with Promises, I think you can remove the call to `done()` AND also remove it as an argument to the spec. So, line 78 would be `it('should sort ascending', () => {`